iis/mymodule.cpp

```diff
@@ -507,11 +616,17 @@ CMyHttpModule::OnSendResponse(
     CriticalSectionLock lock{cs};
 	// here we must check if response body processing is enabled
 	//
-	if(!rsc || !rsc->requestRec || rsc->responseBuffer != nullptr || !modsecIsResponseBodyAccessEnabled(rsc->requestRec.get()))
+	if(!rsc || !rsc->requestRec || rsc->responseBuffer != nullptr || !rsc->responseProcessingEnabled)
```

Comment:
This is a consequence of mixed tabs/spaces indentation in this file :(
I propose I can send a follow-up PR that will specifically bring IIS code to uniformity to avoid such confusing misalignments, just so we don't interfere with the meaningful changes in this PR.

Reply:
Fixed indentation in this specific place, will follow up with an indentation PR afterwards.